acceptance/bundle/templates/dbt-sql/output/my_dbt_sql/resources/my_dbt_sql.job.yml

```diff
@@ -9,10 +9,9 @@ resources:
           interval: 1
           unit: DAYS
 
-      email_notifications:
-        on_failure:
-          - [USERNAME]
-
+      #email_notifications:
```

Comment:
We could use this as an opportunity to educate/include a link with more information about notifications.

@juliacrawf-db Do you have a suggestion?

Reply:
I agree it would be nice. I can add it as a follow up.